MAUI/Scheduler/migration.md

```diff
@@ -482,6 +550,129 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
 
 ### SchedulerAppointmentMapping
 
+The following code example, explains how to map the custom appointments data in Xamarin SfSchedule and .NET MAUI SfScheduler.
+
+<table>
+<tr>
+<th>Xamarin SfSchedule</th>
+<th>.NET MAUI SfScheduler</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<syncfusion:SfSchedule x:Name="schedule">
```

Comment:
syncfusion:SfSchedule to schedule::SfSchedule

Reply:
Updated